File: Sistema/src/app/models/UsuarioLogado.Model.ts
import { PerfilUsuarioListaModel } from "./PerfilUsuarioLista.Model";

export class UsuarioLogadoModel {
  usLCodi: number;
  usuCodi: number;
  usuNome: string;
  lstPerfil: PerfilUsuarioListaModel[];

  constructor() {
    this.usLCodi = 0;
    this.usuCodi = 0;
    this.usuNome = "";
    this.lstPerfil = [];
  }
}

File: API/Models/UsuarioLogadoModel.cs
namespace API_Visitatus.Models
{
    public class UsuarioLogadoModel
    {
        public long usLCodi { get; set; }
        public long usuCodi { get; set; }
        public string? usuNome { get; set; }
        public List<Loja>? lojasUsuario { get; set; }
        public List<PerfilUsuarioListaModel>? lstPerfil { get; set; }
    }
}
